src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  &__minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    transform-origin: bottom center;
+    animation: rotate-hand-minut 3600s infinite linear;
+  }
+
+  &__seconds {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    animation: rotate-hand-second 60s infinite linear;
+    transform-origin: bottom center;
+  }
+
+  &__point {
+    width: 5vmin;
+    height: 5vmin;
+    border-radius: 50%;
+    background-color: #f6a603;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    transform-origin: bottom center;
+  }
+
+  &--speed-up {
+    & .stopwatch__seconds {
+      animation-duration: 10s;
+    }
+
+    & .stopwatch__minutes {
+      animation-duration: 600s;
+    }
+  }
+}
+
+@keyframes rotate-hand-second {
+  from {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+
+  to {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
+
+@keyframes rotate-hand-minut {
```

Comment:
The keyframe animation `rotate-hand-minut` has a typo in its name. Consider renaming it to `rotate-hand-minute` for consistency and to ensure it matches the intended usage in the `.stopwatch__minutes` class.